rqd/rqd/rqcore.py

```diff
@@ -450,6 +468,13 @@ def runDarwin(self):
                                                           self.rqCore.updateRss)
             self.rqCore.updateRssThread.start()
 
+        while True:
+            output = frameInfo.forkedCommand.stdout.readline()
+            if not output and frameInfo.forkedCommand.poll() is not None:
+                break
+            if output:
+                self.rqlog.write(output, prependTimestamp=rqd.rqconstants.RQD_PREPEND_TIMESTAMP)
+
```

Comment:
Similarly to the last comment, the previous code would pipe both stdout and stderr to rqlog, this version is ignoring stderr.

Reply:
And same comment as last :)